addon/components/o-s-s/tag.stories.js

```diff
@@ -0,0 +1,72 @@
+import hbs from 'htmlbars-inline-precompile';
+
+const SkinTypes = ['primary', 'success', 'warning', 'danger', 'secondary'];
+
+export default {
+  title: 'Components/OSS::Tag',
+  component: 'tag',
+  argTypes: {
+    skin: {
+      description: 'Adjust appearance',
+      table: {
+        type: {
+          summary: SkinTypes.join('|')
+        },
+        defaultValue: { summary: 'regular' }
```

Comment:
```suggestion
        defaultValue: { summary: 'primary' }
```
`regular` does not exist in `SkinTypes`